python/rapidsmpf/rapidsmpf/tests/conftest.py

```diff
@@ -14,11 +14,34 @@
 if TYPE_CHECKING:
     from collections.abc import Generator
 
+    from mpi4py import MPI
+
     from rmm.pylibrmm.stream import Stream
 
     from rapidsmpf.communicator.communicator import Communicator
 
 
+def _get_mpi_module_or_skip() -> MPI:
```

Comment:
I'm not sure if it's 100% equivalent, but we could also use `pytest.importorskip("mpi4py")` in any test that needs to import mpi4py. LMK if you want me to look into that.

Reply:
yeah, I think that's a good idea.  I just fixed the import path to get builds passing locally, but I think `importorskip` makes good sense

Reply:
Agree with Mads, if `mpi4py` isn't available we can't do much on the Python end.